client/app/components/composites/Menu/Menu.css

```diff
@@ -0,0 +1,85 @@
+.menu {
+  font-family: var(--topbarFontFamily); /* should come from topbar */
+  font-size: var(--Menu_fontSize);
+  flex-grow: 1;
+
+  &:focus,
+  & :focus {
+    outline: none;
```

Comment:
This is an important accessibility feature. If you remove the outline, something else should indicate the focused element, e.g. when tabbing through links.